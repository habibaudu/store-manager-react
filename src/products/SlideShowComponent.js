import React from 'react';
import {Link} from 'react-router-dom';
import img1 from '../img/jordans shoe.jpeg';
import img2 from  '../img/walltims.jpeg';
import img3 from  '../img/jor.jpeg';
import img4 from  '../img/femaleshoe.jpeg';
import img5 from  '../img/jor2.jpeg';

class SlideShowComponent extends React.Component {
  constructor(props){
    super(props);
  }
  
    render(){
   return (
   <div>
              <nav>
                <div className ='logo'>
                    <h4>Store Manager</h4>
        
                </div>
                        <ul className='nav-links'>
                            <li> <Link to = '/'>Home</Link></li>
                            <li className ='current'> <Link to='/products'>Products</Link></li>
                            <li> <a href ='viewRecord.html'>Records</a></li>
                           
                            
                        </ul>
                                <div className='burger'>
                                    <div className='line1'></div>
                                    <div className='line2'></div>
                                    <div className='line3'></div>
                                </div>
        </nav>
    <header>
       
            <input type="radio" id ='value1' name ='image' readOnly />
            <input type="radio" id ='value2' name ='image' readOnly/>
            <input type="radio" id ='value3' name ='image' readOnly/>
            <input type="radio" id ='value4' name ='image' readOnly/>
            <input type="radio" id ='value5' name ='image' readOnly />
    
            <div className='slide_img' id='one'>

              <img src ={img5} />
              <label className='prev' htmlFor ='value5'>
              </label>
              <label className='next' htmlFor ='value2'>
              </label>
            </div>
        
        <div className='slide_img' id='two'>
            <img src ={img4}/>
            <label className='prev' htmlFor ='value1'>
            </label>
            <label className='next' htmlFor = 'value3'>
            </label>
          </div>
      
    
      <div className='slide_img' id='three'>
        <img src ={img3} />
        <label className='prev' htmlFor ='value2'>
        </label>
        <label className='next' htmlFor ='value4'>
        </label>
      </div>
    
    <div className='slide_img' id='four'>
        <img src ={img2} />
        <label className='prev' htmlFor ='value3'>
        </label>
        <label className='next' htmlFor ='value5'>
        </label>
      </div>
    
    <div className='slide_img' id='five'>
        <img src ={img1} />
        <label className='prev' htmlFor ='value4'>
        </label>
        <label className='next' htmlFor ='value1'>
        </label>
      </div>
    
    <div id='nav_slide'>
    
        <label htmlFor ='value1' className='dots' id='dot1'></label>
        <label htmlFor ='value2' className='dots' id='dot2'></label>
        <label htmlFor ='value3' className='dots' id='dot3'></label>
        <label htmlFor='value4' className='dots' id='dot4'></label>
        <label htmlFor ='value5' className='dots' id='dot5'></label>
    </div>
   
       
    </header>
    
   </div>);
}
};



export default SlideShowComponent;