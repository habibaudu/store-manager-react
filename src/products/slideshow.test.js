import React from 'react';
import { shallow } from 'enzyme';
import SlideShowComponent from './SlideShowComponent';

describe('App', () => {
  it('should render App', () => {
    const component = shallow(<SlideShowComponent />);
    expect(component.exists()).toBe(true);
    expect(component).toMatchSnapshot();
  });
});
