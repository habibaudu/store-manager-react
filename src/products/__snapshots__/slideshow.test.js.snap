// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render App 1`] = `
<div>
  <header>
    <input
      id="value1"
      name="image"
      readOnly={true}
      type="radio"
    />
    <input
      id="value2"
      name="image"
      readOnly={true}
      type="radio"
    />
    <input
      id="value3"
      name="image"
      readOnly={true}
      type="radio"
    />
    <input
      id="value4"
      name="image"
      readOnly={true}
      type="radio"
    />
    <input
      id="value5"
      name="image"
      readOnly={true}
      type="radio"
    />
    <div
      className="slide_img"
      id="one"
    >
      <img
        src={Object {}}
      />
      <label
        className="prev"
        htmlFor="value5"
      />
      <label
        className="next"
        htmlFor="value2"
      />
    </div>
    <div
      className="slide_img"
      id="two"
    >
      <img
        src={Object {}}
      />
      <label
        className="prev"
        htmlFor="value1"
      />
      <label
        className="next"
        htmlFor="value3"
      />
    </div>
    <div
      className="slide_img"
      id="three"
    >
      <img
        src={Object {}}
      />
      <label
        className="prev"
        htmlFor="value2"
      />
      <label
        className="next"
        htmlFor="value4"
      />
    </div>
    <div
      className="slide_img"
      id="four"
    >
      <img
        src={Object {}}
      />
      <label
        className="prev"
        htmlFor="value3"
      />
      <label
        className="next"
        htmlFor="value5"
      />
    </div>
    <div
      className="slide_img"
      id="five"
    >
      <img
        src={Object {}}
      />
      <label
        className="prev"
        htmlFor="value4"
      />
      <label
        className="next"
        htmlFor="value1"
      />
    </div>
    <div
      id="nav_slide"
    >
      <label
        className="dots"
        htmlFor="value1"
        id="dot1"
      />
      <label
        className="dots"
        htmlFor="value2"
        id="dot2"
      />
      <label
        className="dots"
        htmlFor="value3"
        id="dot3"
      />
      <label
        className="dots"
        htmlFor="value4"
        id="dot4"
      />
      <label
        className="dots"
        htmlFor="value5"
        id="dot5"
      />
    </div>
  </header>
</div>
`;
